src/resolvers/company/companyResolver.ts

```diff
@@ -145,12 +148,19 @@ export const createCompanyAdmin = async (parent: never, { firstName, lastName, e
     }
 
     if (!user) {
-      user = await Users.create({ firstName, lastName, email, password, role, address, company });
+      user = await Users.create({ firstName, lastName, role, address, company });
     }
 
     company.admins!.push(user._id);
 
     await company.save();
 
```

Comment:
could this be a utillity that is then used in companyResolver and authResolver instead of duplicate code?